vagrant/src/main/vagrant/servers.yaml

```diff
@@ -41,7 +41,9 @@ servers:
     box: ubuntu/wily64
     ram: 2048
     cpus: 4
-    ip: 10.10.10.100
```

Comment:
@aledsage I've committed a change that enables `autocorrect`, vagrant will attempt to remap the port if it can - it doesn't work for me (I'll need to have a deeper look at why Vagrant refuses to map the port for me), instead throwing a warning that you're trying to bind to a port that's already in use - see the commit for more information on what happens.

I'm inclined to leave it as-is, and just fix the doc reference to the webconsole url [here](http://brooklyn.apache.org/v/0.9.0-SNAPSHOT/start/running.html#control-apache-brooklyn)